enchanted-swiftie/cogs/misc_chat.py

```diff
@@ -0,0 +1,61 @@
+import random
+
+from discord.ext import commands
+from google_images_search import GoogleImagesSearch
+from googleapiclient.errors import HttpError
+
+with open(r"resources\google-token.txt", "rt") as gis_token_file:
+    gis = GoogleImagesSearch(gis_token_file.readline().rstrip(), gis_token_file.readline().strip())
+
+GOOD_ADJECTIVES = ["best", "good", "fantastic", "great"]
+BAD_ADJECTIVES = ["worst", "bad", "trash", "garbage"]
+GOOD_SONGS = ["new romantics", "wildest dreams"]
+
+
+class MiscChat(commands.Cog):
+    def __init__(self, bot: commands.Bot):
+        self.bot = bot
+        self.answers = []
```

Comment:
list of what? add type hints for that, (similar to how you added type hints for dict in the next file)